### DISCUSSION TITLE
Force SSL NextJs without custom nodejs server

### DISCUSSION THREAD

**Discussion by abdulbasit010** [vercel/next.js#20008] [Help]
I just want to redirect my Next js client app to open with `https://` forcefully without using any custom server. the reason is if i use custom node server i'll loose all my Next js optimizations and SEO related things.
Is there any way to redirect my app to `HTTPS `without custom server ?

**Comment by jamesmosier** [comment]
Hi @abdulbasit010. There is no way currently to do this built into Next.js, but this discussion has some potential ideas: https://github.com/vercel/next.js/discussions/10935
